Readme.md

```diff
@@ -10,6 +10,24 @@
 # Description:
 - Social Ranking App. Social Media Platform with profile, individual ranking. Functionality includes ability to rank, search platform, update profile.
 
+# Running:
+- We have dockerized our application to make development easier.
+
+First
+`docker-compose build`
+
+Then
+`docker-compose run web rake db:create`
+
+Finally
+`docker-compose up`
```

Comment:
You can also use `docker-compose up -d`, which will give your terminal prompt back to you, but I prefer not to.